VMBackup/main/Utils/SizeCalculation.py

```diff
@@ -68,7 +73,22 @@ def device_list_for_billing(self):
     def get_total_used_size(self):
         try:
             size_calc_failed = False
-            df = subprocess.Popen(["df" , "-k"], stdout=subprocess.PIPE)
+
+            try:
+                configfile='/etc/azure/vmbackup.conf'
```

Comment:
same, can we use util methods, similar to below:

https://github.com/Azure/azure-linux-extensions/blob/a81f31479ceaf53742eaeb9fe16af8c9a29b3203/VMBackup/main/freezesnapshotter.py#L84

Reply:
Using existing method